packages/client/src/containers/CreateTeamPage/CreateTeam.js

```diff
@@ -0,0 +1,177 @@
+      alert('The member already exist in the team');
+      return;
+    }
+
+    if (!firstName || !lastName || !email) {
+      // eslint-disable-next-line no-alert
+      alert('Please input first name, last name & email');
+    } else {
+      const newMember = {
+        id: null,
+        firstName,
+        lastName,
+        email,
+      };
+
+      teamMembers.push(newMember);
+
+      setTeamMembers([...teamMembers]);
+      setFirstName('');
+      setLastName('');
+      setEmail('');
+    }
+  };
+
+  const addExistingMember = (e) => {
+    const existingTeamMember = teamMembers.find(
+      (member) => member.email === e.target.value,
+    );
+
+    if (existingTeamMember) {
+      // eslint-disable-next-line no-alert
+      alert('The member already exist in the team');
+    } else {
+      const memeber = employees.find(
+        (member) => member.email === e.target.value,
+      );
+
+      teamMembers.push(memeber);
+      setTeamMembers([...teamMembers]);
+    }
+  };
+
+  const handleDelete = (emailToDelete) => {
+    const newTeamMembers = teamMembers.filter(
+      (member) => member.email !== emailToDelete,
+    );
+
+    setTeamMembers(newTeamMembers);
+  };
+
+  return (
+    <div className="create-team">
+      <h1>Create New Team</h1>
+      <form onSubmit={handleSubmit}>
+        <label>Team Name</label>
+        <input
+          type="text"
+          name="teamName"
+          placeholder="Team Name"
+          value={teamName}
+          onChange={(e) => setTeamName(e.target.value)}
+          required
+        />
+
+        <button type="submit">Create Team</button>
+        <br />
+        <h2>Members</h2>
+        <br />
+
+        <div>
+          <label>First Name</label>
+          <input
+            type="text"
+            name="firstName"
+            placeholder="First Name"
+            value={firstName}
+            onChange={(e) => handleInputChange('firstNameInput', e)}
+          />
+          <label>Last Name</label>
+          <input
+            type="text"
+            name="lastName"
+            placeholder="Last Name"
+            value={lastName}
+            onChange={(e) => handleInputChange('lastNameInput', e)}
+          />
+          <br />
+          <label>Email</label>
+          <input
+            type="email"
+            name="email"
+            placeholder="Email"
+            value={email}
+            onChange={(e) => handleInputChange('emailInput', e)}
+          />
+          <br />
+        </div>
+        <label>Select an employee</label>
+        <select onChange={(e) => addExistingMember(e)}>
+          <option value="">Select an existing employee</option>
+          {employees.map((employee) => (
+            <option key={employee.id} value={employee.email}>
+              {employee.firstName} {employee.lastName}
+            </option>
+          ))}
+        </select>
+        <button type="button" onClick={handleAddNewMember}>
```

Comment:
You could do it this way:
```suggestion
        <button type="button" onClick={handleAddNewMember} disabled={!teamMembers.length}>
```